### ISSUE TITLE
fetchInit not working

### ISSUE THREAD

**Issue by enrique-ramirez** [evert/ketting#53]
Hi @evert!

I was working on some other parts of my project until today, where I installed a fresh copy of ketting (v 8.0.1) and tried to fetch some data with `credentials: 'include'`, but it seems that `fetchInit` is not being sent to `fetch`.

Here's a quick slice of my code, maybe I'm doing something wrong?

```javascript
// api.js
import Ketting from 'ketting'

const options = {
  fetchInit: {
    credentials: 'include',
  },
}

const api = new Ketting('https://dev.lvh.me/api', options)

export default api
```

```javascript
// somewhereElse.js
import api from './api'

console.log(api)
api.follow('someResource')
```

The `console.log` shows me the Ketting object, where I can see the fetchInit property (screenshot attached), but once the `follow` runs, no cookies are sent.

![screen shot 2017-09-07 at 4 50 34 pm](https://user-images.githubusercontent.com/1190640/30186818-b4ccd490-93ec-11e7-813d-bb0ef2c26783.png)

If I open my console and run:

```javascript
fetch('https://dev.lvh.me/api', {
    credentials: 'include',
    headers: {
        accept: 'application/json'
    }
})
```

Things work as expected and I can see the cookies being sent.

I'm not sure if I'm going crazy, but I swear I saw this working a couple of days ago.

**Comment by evert** [comment]
hi @enrique-ramirez , 

It looks like I didn't test every case correctly. It now sometimes works, and sometimes doesn't. I just added a fix to the master branch, would you mind taking another look?

**Comment by evert** [comment]
Note, just released 0.8.3, which I think should fix this. Closing this ticket. If I'm wrong, let me know!

**Comment by enrique-ramirez** [comment]
Hi @evert!

Thanks for the update. I tried it again but no dice. The setup and behaviour are exactly the same as in my original post. I've verified that I have v0.8.3 as well, but the `follow` still triggers without sending cookies. I tried with `fetch` for the same results.
